client/src/components/hero_location/heroLocationCtrl.js

```diff
@@ -42,6 +44,33 @@
       });
     };
 
+    vm.callback = function(map) {
+      console.log('callback was called');

```

Comment:
Remove console.log if you won't be using it for testing anymore. If you are going to be using it, you can leave it in. Same for console.logs on line 60 and 65